docs/commands.html

```diff
@@ -0,0 +1,246 @@
+    command /test-command:<br>
+    &nbsp;&nbsp;&nbsp;&nbsp;trigger: <br>
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;broadcast "Command: /test-command"<br>
+    <br>
+    command nickname:<br>
+    &nbsp;&nbsp;&nbsp;&nbsp;trigger: <br>
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;broadcast "Command: /nickname"<br>
+    <br>
+    command //set!:<br>
+    &nbsp;&nbsp;&nbsp;&nbsp;trigger: <br>
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;broadcast "Command: //set!"<br>
+    </div>
+    
+    
+    
+    
+    <h2 class="title">Arguments</h2>
+    <p class="subtitle">
+        <a href="/expressions.html#ExprArgument">Arguments</a> follow the command name, and are separated by spaces. Arguments can be any text you'd like. <br>You can make arguments optional by surrounding them with brackets, like this: <code>command /kill [all entities]</code>.
+        <br>However, the real power in arguments comes from type arguments. These allow a command to take in a <a href="/classes.html">type</a>, like <code>command /kill &lt;player&gt;</code>. The player argument can now be
+        referenced in the trigger section by <code>arg-1</code> or <code>argument 1</code> or a <a href="/expressions.html#ExprArgument">number of other ways</a>. You can also name type variables like so: <code>&lt;name:type&gt;</code>, which can then be refenced as a local variable: <code>{_name}</code>. Here's the full pattern for arguments:
+    </p> 
+    <div class="box skript-code-block left-margin">
+        &lt;name:type=%default value%&gt;
+    </div>
+    <p class="subtitle">
+        Arguments can be used in a lot of different ways, so I'll provide some examples ranging from the simplest possible to more complex uses.
+   </p>
+   <div class="box skript-code-block left-margin">
+        # this command can be run by "/test" or by "/test command".<br>
+        command /test [command]:<br>
+        &nbsp;&nbsp;&nbsp;&nbsp;trigger: <br>
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;broadcast "Command: /test or /test command"<br>
+        <br>
+        # broadcasts the name of the given player. "of" is optional.<br>
+        command /name [of] &lt;player&gt;:<br>
+        &nbsp;&nbsp;&nbsp;&nbsp;trigger: <br>
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;broadcast "Player's name: %arg-1%, %argument 1%, or %player-argument%"<br>
+        <br>
+        # heals the player in the argument. If no player is given, it will heal the sender.<br>
+        command /heal [&lt;player=%player%&gt;]:<br>
+        &nbsp;&nbsp;&nbsp;&nbsp;trigger: <br>
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;heal %argument 1%<br> 
+        <br>
+        
+        # this can be used like /kill zombies, /kill creepers and animals in radius 100 or /kill monsters in the radius 6. <br> # the radius will default to 20 if isn't entered.<br>
+        command /kill &lt;entity types&gt; [in [the] radius &lt;number = 20&gt;]:<br>
```

Comment:
I'd like to avoid introducing anything in the examples that hasn't already been explained. Do you think it's worth changing this example over or could it be left as is?

Reply:
I think it's better to be added with a comment like `will be explained later in the tutorial`